src/cascadia/LocalTests_TerminalApp/TerminalApp.LocalTests.vcxproj

```diff
@@ -43,7 +43,8 @@
     <ClCompile Include="CommandlineTest.cpp" />
     <ClCompile Include="SettingsTests.cpp" />
     <ClCompile Include="TabTests.cpp" />
-	<ClCompile Include="FilteredCommandTests.cpp" />
+    <ClCompile Include="TrustCommandlineTests.cpp" />
+	  <ClCompile Include="FilteredCommandTests.cpp" />

```

Comment:
what's with the indent?

Reply:
what _is_ with the indent here indeed? I bet sublime opened this with tabs set to 2, and I added two spaces to align this, not realizing there was a tab there.